src/components/radio/radio-button.scss

```diff
@@ -0,0 +1,138 @@
+@import "default-theme";
+
+$md-radio-width: 20px !default;
+
+// Top-level container for entire button.
+md-radio-button {
+  display: inline-block;
+  cursor: pointer;
+  padding: 8px;
+  position: relative;
+  white-space: nowrap;
+}
+
+// Container for radio circles and ripple.
+.md-radio-container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: $md-radio-width;
+  position: relative;
+  top: 2px;
+  width: $md-radio-width;
+}
+
+// TODO(mtlin): Replace when ink ripple component is implemented.
+// A placeholder ink ripple, shown when keyboard focused.
+.md-ink-ripple {
+  background: #000;
+  border-radius: 50%;
+  height: 48px;
+  left: 10px;
+  opacity: 0;
+  pointer-events: none;
+  position: absolute;
+  top: 10px;
+  transform: translate(-50%,-50%);
+  transition: opacity ease 0.28s, background-color ease 0.28s;
+  width: 48px;
+  overflow: hidden;
+}
+
+// The outer circle for the radio, always present.
+.md-radio-outer-circle {
+  border: solid 2px;
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $md-radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: border-color ease 0.28s;
+  width: $md-radio-width;
+}
+
+// The inner circle for the radio, shown when checked.
+.md-radio-inner-circle {
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $md-radio-width;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: transform ease 0.28s, background-color ease 0.28s;
+  transform: scale(0);
+  width: $md-radio-width;
+
+  .md-radio-checked & {
+    transform: scale(0.5);
+  }
+}
+
+// Text label next to radio.
+.md-radio-label {
+  display: inline-block;
+  float: right;
+  line-height: 24px;
+  // Equal padding on both sides, for RTL.
+  padding-left: 8px;
+  padding-right: 8px;
+  position: relative;
+  vertical-align: top;
+}
+
+// Underlying native input element.
+// Visually hidden but still able to respond to focus.
+.md-radio-input {
+  position: absolute;
+  width: 0;
+  height: 0;
+  margin: 0;
+  padding: 0;
+  opacity: 0;
+  appearance: none;
+  border: none;
+}
+
+
+// THEME
```

Comment:
The theme stuff doesn't have to be separated from the rest of the css; this was an artifact left over from when I copied and pasted the styles from the ng1 version (where they are separated).